package.json

```diff
@@ -42,12 +44,17 @@
     "@changesets/cli": "^2.26.1",
     "@types/node": "^20.1.2",
     "@wagmi/cli": "^1.0.1",
+    "abitype": "^0.8.7",
     "es-main": "^1.2.0",
     "glob": "^10.2.2",
     "listr2": "^6.4.2",
     "prettier": "^2.8.8",
     "prettier-plugin-solidity": "^1.1.3",
     "tsup": "^6.7.0",
-    "typescript": "^5.0.4"
-  }
+    "typescript": "^5.0.4",
+    "viem": "^1.6.0",
+    "vite": "^4.1.4",
+    "vitest": "~0.30.1"
+  },
+  "packageManager": "pnpm@7.9.4"
```

Comment:
Can we use pnpm here?

Reply:
that shouldn't have been added...will remove

Reply:
removed